ui/app/css/itcss/components/send.scss

```diff
@@ -347,23 +348,23 @@
   }
 
   &__title {
+    @include H4;
+
     color: $scorpion;
-    font-size: 18px;
-    line-height: 29px;
   }
 
   &__description,
   &__balance-text,
   &__token-symbol {
+    @include H6;
+
     margin-top: 10px;
-    font-size: 14px;
-    line-height: 24px;
     text-align: center;
   }
 
   &__token-balance {
-    font-size: 40px;
-    line-height: 40px;
+    @include H1;
```

Comment:
most of this file seems outdated and unused, earmarking for a later PR to remove anything unused.